packages/labs/src/components/tag-input/_tag-input.scss

```diff
@@ -45,6 +45,11 @@ $ti-icon-padding-large: ($pt-input-height-large - $pt-icon-size-standard) / 2;
     padding: 0 ($input-padding-horizontal - $ti-padding);
     line-height: $tag-height;
 
+    // match padding of other input elements iff no tags are selected
+    &:first-child {
```

Comment:
I suppose we can keep this `:first-child` for now, and change things up later when we move `pt-input-ghost` into core for input group refactor. 👍